tests/Feature/StatsTest.php

```diff
@@ -122,4 +123,33 @@ function ($notification, $channels, $notifiable) {
             }
         );
     }
+
+    /** @test */
+    public function hit_counts_match_week_to_week()
+    {
+        $start = CarbonImmutable::now();
+
+        // create one or more hits per day for past 2 weeks
+        while ($start >= CarbonImmutable::now()->subWeeks(2)) {
+            Hit::factory()
+                ->count(mt_rand(1, 4))
+                ->create([
+                    'created_at' => $start,
+                ]);
+
+            $start = $start->subDay();
+        }
```

Comment:
I was trying to think about how we can get different times of day, and one idea I had was:

```suggestion
        // create one or more hits per day for past 2 weeks
        while ($start >= CarbonImmutable::now()->subWeeks(2)) {
            Hit::factory()
                ->create([
                    'created_at' => $start,
                ]);

            $start = $start->subHours(mt_rand(1, 8));
        }
```

... or something like this.